check_manifest.py

```diff
@@ -897,26 +897,30 @@ def should_use_pep_517():
     return True
 
 
-def build_sdist(tempdir, python=sys.executable):
+def build_sdist(tempdir, python=sys.executable, build_isolation=True):
     """Build a source distribution in a temporary directory.
 
     Should be run with the current working directory inside the Python package
     you want to build.
     """
     if should_use_pep_517():
         # I could do this in-process with
-        #   import pep517.envbuild
-        #   pep517.envbuild.build_sdist('.', tempdir)
+        #   import build.__main__
+        #   build.__main__.build('.', tempdir)
         # but then it would print a bunch of things to stdout and I'd have to
         # worry about exceptions
-        run([python, '-m', 'pep517.build', '--source', '-o', tempdir, '.'])
+        cmd = [python, '-mbuild', '--sdist', '.', '--outdir', tempdir]
+        if not build_isolation:
+            cmd.append('--no-isolation')
```

Comment:
This is the only line of code not covered by tests.  A copy of [`Tests.test_build_sdist`](https://github.com/mgedmin/check-manifest/blob/8cbb958d28157cd164932867c2398218e1511d16/tests.py#L609-L625) could do that; call it `test_build_sdist_no_isolation`.

Hmm I should've named that test `test_build_sdist_pep517` rather than just `test_build_sdist`.  Oh well.